pkg/controller/propagator/propagation.go

```diff
@@ -210,6 +201,89 @@ func (r *ReconcilePolicy) handleRootPolicy(instance *policiesv1.Policy) error {
 	return nil
 }
 
+// getApplicationPlacementDecisions return the placement decisions from an application
+// lifecycle placementrule
+func getApplicationPlacementDecisions(c client.Client, pb policiesv1.PlacementBinding, instance *policiesv1.Policy) ([]appsv1.PlacementDecision, *policiesv1.Placement, error) {
+	plr := &appsv1.PlacementRule{}
+	err := c.Get(context.TODO(), types.NamespacedName{Namespace: instance.GetNamespace(),
+		Name: pb.PlacementRef.Name}, plr)
+	if err != nil && !errors.IsNotFound(err) {
+		log.Error(err, "Failed to get plr...", "Namespace", instance.GetNamespace(), "Name",
+			pb.PlacementRef.Name)
+		return nil, nil, err
+	}
+	// plr found, add current plcmnt to placement
+	placement := &policiesv1.Placement{
+		PlacementBinding: pb.GetName(),
+		PlacementRule:    plr.GetName(),
```

Comment:
This code is reached if it wasn't found because of `if err != nil && !errors.IsNotFound(err) {`. Was that intentional?

Reply:
It doesn't seem right does it.  Trying a fix here too

Reply:
@gparvin what did you decide on this comment?